### PR TITLE
Added hw_7.py

### PR DESCRIPTION
**Pull request by Oleksiy-Yurchenko**
Added home work "Classes". Please review.

### CODE REVIEW

File: hw_7.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+# 4) Получить путь относительный путь к файлу
+# 5) Получить абсолютный путь к файлу
+
+class JsonFileHandler:
+
+    def __init__(self, file):
+        if os.path.exists(file) and file.endswith('.json'):
+            self.file = file
+        else:
+            raise ValueError('Class Instance attribute should be a path to a JSON file.')
```
Comment (katevoskresenska) [inline hw_7.py:71]: It would be nice to add json validation to catch JSONDecodeError

File: hw_7.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        with open(self.file, 'r') as file:
+            return json.load(file)
+
+    def write_json(self, data):
+        with open(self.file, 'w') as file:
+            return json.dump(data, file)
+
+    def merge_files(self, other_json_file, output_file):
+        with open(self.file, 'r') as file:
+            data_1 = json.load(file)
```
Comment (mmykhailenko) [inline hw_7.py:83]: use self.read_json

File: hw_7.py [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+        if os.path.exists(other_json_file) and other_json_file.endswith('.json'):
+            with open(other_json_file, 'r') as file:
+                data_2 = json.load(file)
```
Comment (mmykhailenko) [inline hw_7.py:86]: use self.write_json

File: hw_7.py [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+class JsonFileHandler:
+
+    def __init__(self, file):
+        if os.path.exists(file) and file.endswith('.json'):
+            self.file = file
+        else:
+            raise ValueError('Class Instance attribute should be a path to a JSON file.')
+
+    def read_json(self):
+        with open(self.file, 'r') as file:
```
Comment (mmykhailenko) [inline hw_7.py:74]: could be FileNotFound error

**Review by mmykhailenko — APPROVED** [review]

**Pull request merged**